atomic_reactor/__init__.py

```diff
@@ -32,6 +32,8 @@ def format(self, record):
         if not hasattr(record, 'arch'):
             record.arch = '-'
 
+        record.msg = record.msg.replace('\n', ' ')
```

Comment:
Possibly wrap this with 'if not isinstance(record.msg, dict):' or similar? I'm not sure we really want to strip out the newline here though, some of the log parsing and demultiplexing actually hunts for multi-line entries and properly breaks them up, so pulling out the newlines here could lead to problems. Completely untested though.